cms/djangoapps/contentstore/toggles.py

```diff
@@ -22,31 +22,26 @@
 WAFFLE_NAMESPACE = 'contentstore'
 WAFFLE_FLAG_NAMESPACE = LegacyWaffleFlagNamespace(name=WAFFLE_NAMESPACE, log_prefix='Contentstore: ')
 
-# Waffle flag to split library to new view.
 # .. toggle_name: split_library_on_studio_dashboard
 # .. toggle_implementation: WaffleFlag
 # .. toggle_default: False
-# .. toggle_description: Studio dashboard
+# .. toggle_description: Enables data new view for library on studio dashboard.
 # .. toggle_use_cases: open_edx
 # .. toggle_creation_date: 2020-07-8
-# .. toggle_target_removal_date: None
```

Comment:
**Reviewer Note:** This PR should be removing `toggle_target_removal_date: None` in cases where `toggle_use_cases` does **not** include `temporary`. I think they are all the correct cases, but you don't need to spend a lot of time reviewing that to make sure it is perfect. It is an improvement that should help with the copy/paste contagion until we get some additional linting to give us what we want.